src/engine/dispatch.h

```diff
@@ -171,8 +171,9 @@ enum DivDispatchCmds {
   DIV_CMD_X1_010_AUTO_ENVELOPE,
   DIV_CMD_X1_010_SAMPLE_BANK_SLOT,
 
-  DIV_CMD_WS_SWEEP_TIME,
-  DIV_CMD_WS_SWEEP_AMOUNT,
+  DIV_CMD_WS_SWEEP_TIME, // (time)
+  DIV_CMD_WS_SWEEP_AMOUNT, // (value)
+  DIV_CMD_WS_GLOBAL_SPEAKER_VOLUME, // (multiplier)
```

Comment:
Move this to the end. Commands must be stable across versions.

Reply:
Done